### ISSUE TITLE
XenoChron2 changes file size

### ISSUE THREAD

**Issue by ghost** [thinktech2go/XenoChron2#1]
Hello, love the software. One issue I've found is that XenoChron2 changes the file size from 1,180 KB to 1,118 KB, even if I don't change any values. I've enclosed my save file below if you want to take a look at it.

[bf2savefile.zip](https://github.com/thinktech2go/XenoChron2/files/2113856/bf2savefile.zip)

**Comment by type0dev** [comment]
That is a issue, wonder if this is reason why new dlc characters being
screwy.

I will look into this

**Comment by type0dev** [comment]
Can you try version 1.0 and lmk ic it does the same

**Comment by ghost** [comment]
Yeah, it does. The older save files have the 1,118 KB file size, so I'm pretty sure the new data is for DLC blades.

**Comment by mrayburn** [comment]
Any updates on this with the new release (1.1)?
I haven't tried it yet, but I'm just glad I can get the editor to run now :)

**Comment by type0dev** [comment]
What is the exact save file sizes of new dlc


Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Mark <notifications@github.com> Date: 7/16/18  2:21 PM  (GMT-05:00) To: thinktech2go/XenoChron2 <XenoChron2@noreply.github.com> Cc: Type_0_Dev <thinktech2go@gmail.com>, Comment <comment@noreply.github.com> Subject: Re: [thinktech2go/XenoChron2] XenoChron2 changes file size (#1) 
Any updates on this with the new release (1.1)?

I haven't tried it yet, but I'm just glad I can get the editor to run now :)

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/thinktech2go/XenoChron2","title":"thinktech2go/XenoChron2","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/thinktech2go/XenoChron2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@mrayburn in #1: Any updates on this with the new release (1.1)?\r\nI haven't tried it yet, but I'm just glad I can get the editor to run now :)"}],"action":{"name":"View Issue","url":"https://github.com/thinktech2go/XenoChron2/issues/1#issuecomment-405328406"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/thinktech2go/XenoChron2/issues/1#issuecomment-405328406",
"url": "https://github.com/thinktech2go/XenoChron2/issues/1#issuecomment-405328406",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [thinktech2go/XenoChron2] XenoChron2 changes file size (#1)",
"sections": [
{
"text": "",
"activityTitle": "**Mark**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@mrayburn",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"thinktech2go/XenoChron2\",\n\"issueId\": 1,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"thinktech2go/XenoChron2\",\n\"issueId\": 1\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/thinktech2go/XenoChron2/issues/1#issuecomment-405328406"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 347300179\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by mrayburn** [comment]
I won't be back home to get this to you until late tonight.

**Comment by type0dev** [comment]
Will fix today or tomorrow, thanks

**Comment by mrayburn** [comment]
This is my current save. With DLC bought, and having met (but not permanently kept) Shulk and Fiora.

[save.zip](https://github.com/thinktech2go/XenoChron2/files/2201958/save.zip)
